docs/readmes/lte/deploy_install_debian.md

```diff
@@ -1,40 +1,42 @@
 ---
-id: deploy_install_ubuntu
-title: Install AGW
+id: deploy_install_debian
+title: Install AGW on Debian (Stretch)
 hide_title: true
 ---
 
-# Install Access Gateway on Ubuntu (Bare Metal)
+For Installing AGW on Ubuntu refer to : [_deploy_on_ubuntu_](https://magma.github.io/magma/docs/lte/deploy_install_ubuntu)
```

Comment:
Add a big fat `> NOTE` at the top of this that this installation target OS will be deprecated (when?)